web/src/components/tooltips/common.js

```diff
@@ -27,6 +27,6 @@ const Flag = styled.img``;
 export const ZoneName = ({ zone, ellipsify }) => (
   <React.Fragment>
     <Flag className="flag" alt="" src={flagUri(zone)} />
-    {ellipsify ? getShortZoneName(zone) : getFullZoneName(zone)}
+    {getZoneNameWithCountry(zone, ellipsify ? 40 : 0)}
```

Comment:
Nitpick: For readability, we might want a number reference for the limit instead of passing '40'. Also we can avoid passing a limit and instead passing undefined, which I also think reads slightly better as passing 0 is the same is saying there's no limit.

So something like this: 

const LIMIT = 40;

getZoneNameWithCountry(zone, ellipsify ? LIMIT : undefined)